packages/dapp-connector/src/WalletApi/Cip30Wallet.ts

```diff
@@ -1,7 +1,63 @@
 import { APIErrorCode, ApiError } from '../errors';
+import { Bytes, Cbor, Paginate, WalletApi, WalletApiExtension, WalletMethod } from './types';
+import { Cardano } from '@cardano-sdk/core';
 import { Logger } from 'ts-log';
 import { RemoteAuthenticator } from '../AuthenticatorApi';
-import { WalletApi } from './types';
+import uniq from 'lodash/uniq';
+
+export const CipMethodsMapping: Record<number, WalletMethod[]> = {
+  30: [
+    'getNetworkId',
+    'getUtxos',
+    'getCollateral',
+    'getBalance',
+    'getUsedAddresses',
+    'getUnusedAddresses',
+    'getChangeAddress',
+    'getRewardAddresses',
+    'signTx',
+    'signData',
+    'submitTx'
+  ],
+  95: ['getActivePubStakeKeys', 'getPubDRepKey']
+};
+export const WalletApiMethodNames: WalletMethod[] = Object.values(CipMethodsMapping).flat();
+
+/**
+ * Wrap the proxy API object with a regular javascript object to avoid interop issues with some dApps.
```

Comment:
It would have been better to commit this as a separate change first as it's considered a fix in some contexts.